project/ProjectPlugin.scala

```diff
@@ -90,38 +81,26 @@ object ProjectPlugin extends AutoPlugin {
         "com.github.ghik"        % "silencer-lib"        % V.silencer  % Provided cross CrossVersion.full,
         compilerPlugin("com.github.ghik" % "silencer-plugin" % V.silencer cross CrossVersion.full)
       ),
-      libraryDependencies ++= (CrossVersion.partialVersion(scalaBinaryVersion.value) match {
-        case Some((2, 13)) => Seq.empty[ModuleID]
-        case _ =>
-          Seq(compilerPlugin("org.scalamacros" %% "paradise" % V.paradise cross CrossVersion.full))
-      })
+      libraryDependencies ++= on(2, 12)(
+        compilerPlugin("org.scalamacros" %% "paradise" % V.paradise cross CrossVersion.full)
+      ).value
     )
 
   }
 
-  import autoImport.V
-
   override def projectSettings: Seq[Def.Setting[_]] =
     Seq(
-      organization := "com.47deg",
-      crossScalaVersions := Seq(V.scala212, V.scala213),
-      scalacOptions := {
-        val withStripedLinter = scalacOptions.value filterNot Set("-Xlint", "-Xfuture").contains
-        (CrossVersion.partialVersion(scalaBinaryVersion.value) match {
-          case Some((2, 13)) => withStripedLinter :+ "-Ymacro-annotations"
-          case _             => withStripedLinter
-        }) :+ "-language:higherKinds"
-      },
+      scalacOptions ++= on(2, 13)("-Ymacro-annotations").value,
       coverageMinimum := 70d,
       coverageFailOnMinimum := true,
-      coverageExcludedPackages := "<empty>;github4s\\.scalaz\\..*",
-      // This is necessary to prevent packaging the BuildInfo with
-      // sensible information like the Github token. Do not remove.
-      mappings in (Compile, packageBin) ~= { (ms: Seq[(File, String)]) =>
-        ms filter {
-          case (_, toPath) =>
-            !toPath.startsWith("github4s/BuildInfo")
-        }
-      }
+      coverageExcludedPackages := "<empty>;github4s\\.scalaz\\..*"
```

Comment:
unrelated to this PR but we can remove that 👍

Reply:
Done in https://github.com/47degrees/github4s/pull/501/commits/2d14932947cf1f51fc6bee242e202e41ecbe211d! Thanks!